helm/presets/local.yaml

```diff
@@ -1,8 +1,8 @@
 receive:
+  service_udp:
+    type: LoadBalancer
```

Comment:
exposing a service locally using docker's built-in k8s exposes the service to the host machine without any extra networking shenanigans.